src/solver/hydro/management/management.cpp

```diff
@@ -368,15 +368,15 @@ void HydroManagement::changeInflowsToAccommodateFinalLevels(uint year)
     {
         auto& data = tmpDataByArea_[&area];
 
-        if (!area.hydro.finalLevelInflowsModifier.isApplicable(year))
+        if (!area.hydro.deltaBetweenFinalAndInitialLevels[year].has_value())
             return;
 
         // Must be done before prepareMonthlyTargetGenerations
-        double delta = area.hydro.finalLevelInflowsModifier.deltaLevel[year];
```

Comment:
Signs (in the sense of +/-) changed because delta is now : final level - initial level, more natural than its opposite.